ingestion-pipeline/helm/values.yaml

```diff
@@ -62,6 +62,8 @@ defaultPipeline:
   # version of the knowledgebase
   version: "1.0"
   vector_db_name: "demo-rag-vector-db-v1-0-s3"
+  # generate provenance for the knowledge base
+  enableSigning: false
```

Comment:
@Roming22 defaultPipeline is the object that is related to a job that triggers an initial pipeline - you can basically install this chart and execute a pipeline immediately without going through the service (it basically just executes a curl command once the service is up).  Lets move it outside.